### PR TITLE
Home page translation

### PR DESCRIPTION
**Pull request by saranshkataria**
Though the home page is not complete, I have translated the 3 sections and I thought of putting this in to break the ice and let people see something in place. Also, this should provide an opportunity to get in some initial feedback, if any. 

@arshadkazmi42 thoughts?

### CODE REVIEW

File: content/home/marketing/learn-once-write-anywhere.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Learn Once, Write Anywhere
+title: एक बार जानें, कहीं भी लिखें
 order: 2
 ---
 
-We don't make assumptions about the rest of your technology stack, so you can develop new features in React without rewriting existing code.
+हम आपके अन्य तकनिकी स्टैक के बारे में धारणा नहीं बनाते हैं। तो आप मौजूदा कोड को फिर से लिखे बिना रिएक्ट में नई सुविधाएँ बना सकते हैं।
```
Comment (arshadkazmi42) [inline content/home/marketing/learn-once-write-anywhere.md]: Are we translating `React` word? 
As you have translated it in this line to `रिएक्ट`
And above file its not translated

File: content/home/marketing/declarative.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Declarative
+title: वर्णनात्मक
 order: 0
 ---
 
-React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes.
+React में इंटरैक्टिव यूआई बनाना काफी आसान है। अपने ऐप में प्रत्येक व्यू के लिए सरल विचार डिजाइन करें, और React कुशलता से आपके डेटा में बदलाव होने पर सिर्फ सही कौम्पोनॅन्ट को अपडेट और रेंडर करेगा।
```
Comment (arshadkazmi42) [inline content/home/marketing/declarative.md]: React word is not translated here and its translated in below file.
Let's keep it consistent. Either we will translate the word or we will keep it as it is.

File: content/home/marketing/learn-once-write-anywhere.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
-React can also render on the server using Node and power mobile apps using [React Native](https://facebook.github.io/react-native/).
+रिएक्ट सर्वर पर भी रेंडर कर सकता है और आप [React Native]((https://facebook.github.io/react-native/) का उपयोग करके पावर मोबाइल ऐप भी बना सकते हैं।
```
Comment (arshadkazmi42) [inline content/home/marketing/learn-once-write-anywhere.md]: `रिएक्ट` translation of React word.

File: content/home/marketing/component-based.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-title: Component-Based
+title: कौम्पोनॅन्ट पर आधारित
 order: 1
 ---
 
-Build encapsulated components that manage their own state, then compose them to make complex UIs.
+इनकैप्सुलेटेड कौम्पोनॅन्ट का निर्माण करें जो उनके अपने व्यू को संभालते हैं, फिर उनसे कठिन यूआई निर्मित करें।
 
-Since component logic is written in JavaScript instead of templates, you can easily pass rich data through your app and keep state out of the DOM.
+चूँकि कौम्पोनॅन्ट तर्क टेम्पलेट के बजाय जावास्क्रिप्ट में लिखे गए हैं, आप आसानी से अपने ऐप के माध्यम से समृद्ध डेटा पास कर सकते हैं और डोम से बाहर रख सकते हैं।
```
Comment (arshadkazmi42) [inline content/home/marketing/component-based.md]: We are not translating DOM word. Let's keep it as it is.

**Comment by saranshkataria** [comment]
@arshadkazmi42 fixed the translation of DOM. Let's not do react for now, I checked other languages and didn't see it being translated anywhere else as well. If we do that, the translated word might have a different meaning (not the case in hindi though, I agree). But let us keep it consistent for now and if we do decide to translate, it should be a single find and replace all?

File: content/home/marketing/learn-once-write-anywhere.md (hunk 1 above)
Comment (arshadkazmi42) [reply content/home/marketing/learn-once-write-anywhere.md]: @saranshkataria I think you missed one word here.
You have translated react in this line.

**Pull request merged**